modules/pruning-deployments.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * masters/pruning-objects.adoc
+
+[id='pruning-deployments-{context}']
+= Pruning deployments
+
+In order to prune deployments that are no longer required by the system due to
+age and status, administrators can run the following command:
+
+----
+$ oc adm prune deployments [<options>]
+----
+
+.Prune deployments CLI configuration options
+[cols="4,8",options="header"]
+|===
+
+|Option |Description
+
+.^|`--confirm`
+|Indicate that pruning should occur, instead of performing a dry-run.
+
+.^|`--orphans`
+|Prune all deployments whose Deployment Configuration no longer exists, status is
```

Comment:
The usage of "whose" in these descriptions throws me off since it personifies "deployments"; maybe update to "in which" or something similar.